README.md

```diff
@@ -2652,7 +2652,7 @@ Unit testing helps maintain clean code, as such I included some of my recommenda
           /**
            * @name logError
            * @desc Logs errors
-           * @param {String} msg Message to log
+           * @param {String} msg - Message to log.
            * @returns {String}
```

Comment:
These should probably be changed to `string` (lowercase), to match what is said below.

(Also, I'm not a fan of using the hyphen in the `@param` description, but that's entirely personal preference, and I'm not bothered either way; I'm interested to hear the opinions of others on this)